drivers/bridge/bridge.go

```diff
@@ -213,6 +213,8 @@ func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
 		} else {
 			return types.BadRequestErrorf("invalid type for EnableIPTables value")
 		}
+	} else {
```

Comment:
Please set these values to the default in the beginning of the function rather than as an else clause and if there is a difference in caller's config it will be changed anyways when we parse them and react to them.

As a side note this function has to be refactored. Too many indentations and if/else clauses.